### PR TITLE
Capturing preflight checks timings

### PR DESCRIPTION
**Pull request by morenod**
- Capturing preflight timings and save them on ES
- Modify the way we are capturing Namespace creation timing, from sequential (service cluster > management cluster) to parallel including the preflights
- Modify kubeconfig file downloaded according with the last information about removing the CA

### CODE REVIEW

**Review by krishvoor — APPROVED** [review]
/lgtm

File: rosa-hypershift/rosa-hosted-wrapper.py [hunk 1]
```diff
@@ -441,6 +445,42 @@ def _download_cluster_admin_kubeconfig(rosa_cmnd, cluster_name, my_path):
     return return_data
 
 
+def _preflight_wait(rosa_cmnd, cluster_id, cluster_name):
+    logging.info('Collecting preflight times for cluster %s' % cluster_name)
+    return_data = {}
+    start_time = int(time.time())
+    previous_status = ""
+    for trying in range(1, 101):
```
Comment (dry923) [inline rosa-hypershift/rosa-hosted-wrapper.py]: @morenod can we bump this up to 150 attempts? that would give us 5 minutes of timeout

Comment (morenod) [reply rosa-hypershift/rosa-hosted-wrapper.py]: Changed to 150 retries

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**